### ISSUE TITLE
jCenter release doesn't work

### ISSUE THREAD

**Issue by hzsweers** [aalmiray/markdown-gradle-plugin#16]
Appears to not be able to find the Remark dependency

```
Error:Could not find com.overzealous:remark:0.9.3.
Searched in the following locations:
    file:/Applications/Android Studio.app/Contents/gradle/m2repository/com/overzealous/remark/0.9.3/remark-0.9.3.pom
    file:/Applications/Android Studio.app/Contents/gradle/m2repository/com/overzealous/remark/0.9.3/remark-0.9.3.jar
    https://jcenter.bintray.com/com/overzealous/remark/0.9.3/remark-0.9.3.pom
    https://jcenter.bintray.com/com/overzealous/remark/0.9.3/remark-0.9.3.jar
    https://maven.fabric.io/public/com/overzealous/remark/0.9.3/remark-0.9.3.pom
    https://maven.fabric.io/public/com/overzealous/remark/0.9.3/remark-0.9.3.jar
    https://raw.github.com/xujiaao/mvn-repository/master/releases/com/overzealous/remark/0.9.3/remark-0.9.3.pom
    https://raw.github.com/xujiaao/mvn-repository/master/releases/com/overzealous/remark/0.9.3/remark-0.9.3.jar
Required by:
    :app:unspecified > org.kordamp:markdown-gradle-plugin:1.0.0
```

It does appear to work fine if your private bintray repo is specified though

```
maven {
    // For the markdown plugin
    url 'http://dl.bintray.com/content/aalmiray/kordamp'
}
```

**Comment by drazzib** [comment]
+1

I can confirm that I'm unable to use plugins as deployed on plugins.gradle.org
https://plugins.gradle.org/plugin/org.kordamp.gradle.markdown

**Comment by scholarsmate** [comment]
+1

Remark is a library for taking (X)HTML input and outputting markdown.

I think the vast majority of the plugin users need the markdown to HTML conversion rather than the other way around.  Remark doesn't appear to be actively maintained.

Why not have the plugin focus on the markdown to HTML conversion (and jettison the markdown to HTML conversion) and if there is a desire to convert HTML into markdown, then publish a _seperate_ plugin for handling that task?

**Comment by martinda** [comment]
Any news on this? Using the private bintray does not seem to work for me.

**Comment by martinda** [comment]
Looks like "remark" version 1.0.0 was pushed to jcenter:
https://bitbucket.org/OverZealous/remark/issues/4/publish-artifacts-to-maven-ivy-repo#comment-20898525

So I have submitted a pull request to upgrade this project.

**Comment by aalmiray** [comment]
Fixed by merging #17
